packages/react-native/scripts/cocoapods/codegen.rb

```diff
@@ -108,3 +108,11 @@ def run_codegen!(
     codegen_utils.generate_react_codegen_podspec!(react_codegen_spec, codegen_output_dir)
   end
 end
+
+def basePath(react_native_path, relative_installation_root)
+  if react_native_path.start_with?("/")
```

Comment:
```suggestion 
  if Pathname.new(react_native_path).absolute
```

This suggestion also need a `require 'pathname'` after the license.
(See [here](https://stackoverflow.com/a/2316795) for reference)